com.eminfedar.vaktisalah-gtk.json

```diff
@@ -0,0 +1,101 @@
+{
+    "id": "com.eminfedar.vaktisalah-gtk",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "vaktisalah-gtk",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "libsass",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/libsass.git",
+                    "branch": "meson"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
+            "buildsystem":"meson",
+            "builddir": true,
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://github.com/lazka/sassc.git",
+                    "branch": "meson"
+                }
+            ]
+        },
+        {
+            "name": "libadwaita",
+            "buildsystem":"meson",
+            "config-opts" : [
+                "-Dvapi=false",
+                "-Dexamples=false",
+                "-Dtests=false"
+            ],
+            "sources": [
+                {
+                    "type":"git",
+                    "url":"https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "tag": "1.3.2"
+                }
+            ]
+        },
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+              {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                "tag": "v0.6.0"
+              }
+            ]
+        },
+        {
+            "name": "vaktisalah-gtk",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/eminfedar/vaktisalah-gtk-rs",
+                    "tag": "v1.0.0"
+                  }
+            ],
+            "build-options": {
+                "append-path" : "/usr/lib/sdk/rust-stable/bin",
+                "build-args": [
+                    "--share=network"
+                ],
+                "env" : {
+                    "CARGO_HOME" : "/run/build/vaktisalah-gtk/.cargo",
+                    "LOCALIZATION_DIR": "../localization/"
+                }
+            },
+            "build-commands": [
+                "cargo build --release --target x86_64-unknown-linux-gnu",
```

Comment:
so what happen when the flatpak is built on aarch64?

Reply:
why do you specify a target in the first place?

Reply:
To copy the result of the compilation to `/app/bin` as I see in examples:
```
                         |
                         V
"install -Dm755 ./target/x86_64-unknown-linux-gnu/release/vaktisalah-gtk-rs -t /app/bin/"
```

Reply:
so you don't need.